mmv1/third_party/terraform/data_sources/data_source_sql_database_instance_list.go

```diff
@@ -0,0 +1,195 @@
+package google
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	sqladmin "google.golang.org/api/sqladmin/v1beta4"
+)
+
+func dataSourceSqlDatabaseInstanceList() *schema.Resource {
+
+	return &schema.Resource{
+		Read: dataSourceSqlDatabaseInstanceListRead,
```

Comment:
For datasources that request information about many entities the convention is to use a pluralised version of the resource name, e.g. we have a [folder](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/folder) and a [folders](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/folders) data source.

Could you please replace the references to `list` with a pluralised version of the resource, including the name of this file?

Reply:
Thanks for addressing this!